modules/standard/AutoGpu.chpl

```diff
@@ -47,6 +103,29 @@ module AutoGpu {
     __primitive("chpl_assert_on_gpu");
   }
 
+  /*
+   This attribute can be applied to loops and variable declarations to specify
```

Comment:
ditto loop vs variable declaration (the more I read it, the more I find it unnecessary)